### DISCUSSION TITLE
prisma db seed in production docker pipeline

### DISCUSSION THREAD

**Discussion by jasonsilvers** [prisma/prisma#6352] [Q&A]
We are deploying an application and in our docker file we are running:

prisma migrate deploy - this works
prisma db seed - this errors and says, Unknown command "seed"

I am guessing that seed is not available in a production environment? Am I correct in assuming that?

If that is correct, how do you seed a database in production?

Thanks.

**Comment by ryands17** [comment]
@jasonsilvers 👋 

Seeding is still a [preview feature](https://www.prisma.io/docs/guides/application-lifecycle/seed-database) so you need to append `--preview-feature` to `prisma db seed`.

**Reply by jasonsilvers** [reply]
Thanks for the response. I tried it with --preview-feature and it is throwing the same error. 

Do you know in general what would cause the error? Is it simply the binary is missing (This doesn't make sense because prisma migrate deploy works)? Are there any cases where prisma migrate deploy would work but not prisma db seed? 

Thanks for the help.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
